src/index.html

```diff
@@ -4,11 +4,226 @@
+     <td text-align:right;>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a great place for you to tell a story and let your users know a little more about you.</td>
+     </tr>
+     </table>
+     <div class="carousel-caption"> 
+     </div>
+     </div>
+     </div>
+      <div class="item">
+          <h2>EXPERIENCE</h2>
+          <div class="w3-row-padding">
+          <div class="w3-third">
+          <h3>Budget Director</h3>
+          <a href="https://www.wix.com"> @Wix.com </a>
+          <h3>2014 - Current</h3>
+          <p >This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly</p>
+          </div>
+          <div class="w3-third">
+          <h3>Business Administration</h3> 
+          <a  href="https://www.wix.com"> @BizzNizz.net </a>
+          <h3>2010 - 2003</h3>
+          <p>This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly</p>
+          </div>
+          <div class="w3-third">
+          <h3>Finance Director</h3>
+          <a  href="https://www.wix.com"> @Computers & More LTD. </a>
+          <h3>2008 - 2012</h3>
+          <p>This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly</p>
+          </div>
+      </div>
+      </div>
+      <div class="item">
+        <h2>EDUCATION</h2>
+        <div class="w3-row-padding">
+        <div class="w3-third">
+        <h3>The San Francisco Business School</h3>
+        <h3>2010 - 2012</h3>
+        <p>This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.</p>
+        </div>
+        <div class="w3-third">
+        <h3>NYC School of Marketing and Advertising</h3>
+        <h3>2007 - 2010</h3>
+        <p>This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.</p>
+        </div>
+        <div class="w3-third">
+        <h3>L.A. University of Applied Sciences</h3>
+        <h3>2002 - 2006</h3>
+        <p>This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.</p>
+        </div>
+        </div>
+      </div>
+      <div class="item">
+        <h2>HOBBIES & INTERESTS</h2>
+        <div class="row">
+        <div class="col"><img width="150" height="150" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSf71rBnViRBrals8RJieIrnGbQj8kNO8X-8cbRNKKDDnaQauOH" /></div>
```

Comment:
missing `alt` attribute in the image tag

Reply:
already added thanks